modules/ROOT/pages/index.adoc

```diff
@@ -11,8 +11,12 @@ The same editor also allows you to edit RAML and WSDL API specification files an
 
 == Requirements
 
-* Internet Explorer versions 11 and earlier are not supported. +
-Anypoint Studio uses your system's native browser to display elements of the Anypoint Platform UI. If your system defaults to an Internet Explorer browser, you won't be able to visualize any of the Anypoint Platform views.
+* Studio 6.x will miss the following features in Windows:
+** Internet Explorer version 11 and earlier are not supported.
+** Anypoint Studio will not display the Anypoint Platform views if your system has a default Internet browser.
```

Comment:
I would change this by something like:

> ... features in Windows due to Internet Explorer is not supported anymore by Anypoint Platform (See https://docs.mulesoft.com/general/browser-support):

and remove these last 2 sub-items because:
- Not a single one IE version is supported anymore
- it gives the sensation that there is a workaround configuring other browser by default, which is not, it depends on the native browser of Windows. 
- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -

Reply:
I've made some changes, but I'm not sure I understood your comments. Please, can you take a look at it again? Thanks!

Reply:
Thanks for the comments!
I've made some changes based on your comments.
If some parts of the web UI will not work in the future, it does not seem efficient to comment about it now.
Please, can you take a look at them and let me know if they are OK? Thanks!